example/bwmf/proto/bwmf.proto

```diff
@@ -19,9 +19,10 @@ message Response {
 }
 
 message MatrixShard {
-  message RowData {
-    int32 rowId = 1;
-    map<int32, float> at = 2;
-  }
-  repeated RowData row = 1;
+  bool is_sparse = 1;
+  uint32 m = 2;
+  uint32 n = 3;
+  repeated float val = 4;
```

Comment:
Specify whether row-major or column-major should be used. We probably should use column-major in order to be friendly with linear algebra libraries.